tests/test-barplots.js

```diff
@@ -298,6 +298,7 @@ require([
         notOk(layer1.colorByFM);
         equal(layer1.colorByFMField, empress._barplotPanel.fmCols[0]);
         equal(layer1.colorByFMColorMap, "discrete-coloring-qiime");
+        equal(layer1.colorByFMColorReverse, false);
```

Comment:
(QUnit's syntax is kinda funky, but the surrounding tests use `notOk()` and `ok()` to assert false/true respectively so this should probs be consistent.)

```suggestion
        notOk(layer1.colorByFMColorReverse);
```